src/Circletracer.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as fs from 'fs';
```

Comment:
Note to use asterisk, should we not use the descriptor(got from `as`) fs? Then, it's gonna be hard to know where the method came from.

Reply:
Unfortunately _named import_ such as `import { some_name} from somewhere` does not seem to allow `as`. T.T

https://www.typescriptlang.org/docs/handbook/modules.html#import